bindings/experimental/distrdf/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 1995-2021, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+###########################################################
+# CMakeLists.txt file for building DistRDF
+###########################################################
+
+set(DistRDFDir python/DistRDF)
+file(COPY ${DistRDFDir} DESTINATION ${localruntimedir})
```

Comment:
What we usually do with Python files is to build and install the compiled version of them too. You can use as example this:

https://github.com/root-project/root/blob/master/bindings/jupyroot/CMakeLists.txt#L66-L70

And as you can see it's done for every Python version ROOT is built with.

Reply:
I copied this logic from jupyroot over to  CMakeLists.txt but I couldn't get the python sources compiled on the install directory. Still have to figure it out 100%

Reply:
This is now solved in the latest PR